components/director/internal/domain/runtime/resolver.go

```diff
@@ -70,7 +69,11 @@ func (r *Resolver) Runtimes(ctx context.Context, filter []*graphql.LabelFilter,
 
 	ctx = r.ctxvs.WithValue(ctx, persistence.PersistenceCtxKey, tx)
 
-	runtimesPage, err := r.svc.List(ctx, labelFilter, first, &cursor)
+	if first == nil {
+		return nil, errors.New("first is nil, internal server error")
```

Comment:
Return nice error like "first parameter is required". BTW, can't we enforce it on the GraphQL query?

Reply:
Unfortunately not, GraphQL generate this param as pointer :(

Reply:
okay, keep it as it is